index.js

```diff
@@ -23,7 +23,7 @@ function Argv (processArgs, cwd) {
         .map(function (x) {
             // ignore the node bin, specify this in your
             // bin file with #!/usr/bin/env node
-            if (~x.indexOf('node') || ~x.indexOf('iojs')) return;
+            if (/^(node|iojs)\b/.test(x)) return;
```

Comment:
I'm pretty sure there are cases where the bin can be:

`/usr/bin/node`, or `/user/bin/iojs`, perhaps we could modify the regex to something like:

`/(node|iojs)$/`?